doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -220,63 +226,85 @@ Requests to the environment
    .. seealso:: Section :ref:`performingcomputations`.
 
 
-.. cmd:: Print Assumptions @qualid
+.. cmd:: Print Assumptions @smart_qualid
 
-   This commands display all the assumptions (axioms, parameters and
+   Displays all the assumptions (axioms, parameters and
    variables) a theorem or definition depends on. Especially, it informs
    on the assumptions with respect to which the validity of a theorem
    relies.
 
-   .. cmdv:: Print Opaque Dependencies @qualid
-      :name: Print Opaque Dependencies
+   .. todo I think drop the second sentence
 
-      Displays the set of opaque constants :n:`@qualid` relies on in addition to
-      the assumptions.
+.. cmd:: Print Opaque Dependencies @smart_qualid
 
-   .. cmdv:: Print Transparent Dependencies @qualid
-      :name: Print Transparent Dependencies
+   Displays the opaque constants :n:`@smart_qualid` relies on in addition to
+   the assumptions.
 
-      Displays the set of transparent constants :n:`@qualid` relies on
-      in addition to the assumptions.
+   .. todo it printe assumptions+opaque constants?
 
-   .. cmdv:: Print All Dependencies @qualid
-      :name: Print All Dependencies
+.. cmd:: Print Transparent Dependencies @smart_qualid
 
-      Displays all assumptions and constants :n:`@qualid` relies on.
+   Displays the transparent constants :n:`@smart_qualid` relies on
+   in addition to the assumptions.
 
+   .. todo ditto
 
-.. cmd:: Search @qualid
+.. cmd:: Print All Dependencies @smart_qualid
 
-   This command displays the name and type of all objects (hypothesis of
-   the current goal, theorems, axioms, etc) of the current context whose
-   statement contains :n:`@qualid`. This command is useful to remind the user
-   of the name of library lemmas.
+   Displays all the assumptions and constants :n:`@smart_qualid` relies on.
 
-   .. exn:: The reference @qualid was not found in the current environment.
 
-      There is no constant in the environment named qualid.
+.. cmd:: Search @search_item {? {| {| inside | outside } {+ @qualid } | {+ @search_item } } }
```

Comment:
This is incorrect. Probably a mistake in `common.edit_mlg`. The `{| inside | outside }` clause is only supported *after* all the `search_item`.

This should be `Search {+ @search_item } {? {| inside | outside } {+ @qualid } }`.